external-crates/move/documentation/book/src/uses.md

```diff
@@ -69,6 +69,8 @@ fun new_vec(): vector<std::option::Option<u8>> {
 }
 ```
 
+### Multiple aliases
```

Comment:
```suggestion
### Multiple Aliases
```
I believe the rest of the book uses title case for these headings. The following ones should, too.